### PR TITLE
added changes to login signup page

### LINKED ISSUE #660: ✨[Feature Request]: Addition of navbar and footer and improvement in the ui of  signin and signup page that match the theme
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Screenshot 2024-06-21 at 11 09 18 AM
Using html and css the styling should be changed to the required design.

### Use Case

 Addition of navbar and footer and improvement in the ui of  signin and signup page that match the theme.

### Benefits

_No response_

### Add ScreenShots

_No response_

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by DiwareNamrata23**
## Related Issues

- Closes #660 

## Screenshots / videos (if applicable)

Screenshot 2024-08-08 at 10 54 22 PM
Screenshot 2024-08-08 at 10 54 30 PM

## Checklist

- [x] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [x] I have updated my branch and synced it with project `main` branch before making this PR
- [x] I have performed a self-review of my code
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by DiwareNamrata23** [comment]
@varshith257 removed conflicts

File: index.html [hunk 1]
```diff
@@ -91,7 +91,7 @@
         <li><a href="#doctors"><i class="bi bi-person-circle"></i> Doctor</a></li>
         <li><a href="#contact"><i class="bi bi-telephone"></i> Contact us</a></li>
         <li><a href="#feedback"><i class="bi bi-star"></i> Rate Us</a></li>
-        <li><a href="log/login.html"><i class="bi bi-box-arrow-right"></i> Sign in/Sign up</a></li>
+        <li><a href="login.html"><i class="bi bi-box-arrow-right"></i> Sign in/Sign up</a></li>
```
Comment (varshith257) [inline index.html:94]:
```suggestion
        <li><a href="log/login.html"><i class="bi bi-box-arrow-right"></i> Sign in/Sign up</a></li>

```

**Comment by varshith257** [comment]
Also align footer with the home page one

**Comment by DiwareNamrata23** [comment]
Plz merge my pr

**Comment by DiwareNamrata23** [comment]
@varshith257 now there are no conflicts plz merge my pr

**Comment by varshith257** [comment]
@DiwareNamrata23  In deploy the page is misaligned. Have a look

**Comment by DiwareNamrata23** [comment]
Hi @varshith257 can you plz elaborate where is the issue

**Comment by varshith257** [comment]
> ### <span aria-hidden="true">✅</span> Deploy Preview for *rapidoc* ready!
> 
> 
> |  Name | Link |
> |:-:|------------------------|
> |<span aria-hidden="true">🔨</span> Latest commit | 30c407077222b97d5a70386ddfcb974b49d7b583 |
> |<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/rapidoc/deploys/66b591c36a3c150008c3fa3d |
> |<span aria-hidden="true">😎</span> Deploy Preview | [https://deploy-preview-1162--rapidoc.netlify.app](https://deploy-preview-1162--rapidoc.netlify.app) |
> |<span aria-hidden="true">📱</span> Preview on mobile | <details><summary> Toggle QR Code... </summary><br /><br />QR Code<br /><br />_Use your smartphone camera to open QR code link._</details> |
> ---
> 
> _To edit notification comments on pull requests, go to your [Netlify site configuration](https://app.netlify.com/sites/rapidoc/configuration/deploys#deploy-notifications)._

Here

**Comment by DiwareNamrata23** [comment]
@varshith257 can we have a quick google meet?

**Comment by varshith257** [comment]
> @varshith257 can we have a quick google meet?

Sure

**Comment by DiwareNamrata23** [comment]
To join the video meeting, click this link: https://meet.google.com/byb-bhic-eia
Otherwise, to join by phone, dial +1 336-365-5804 and enter this PIN: 209 873 779#

File: log/script.js [hunk 2]
```diff
@@ -5,22 +5,6 @@ document.addEventListener("DOMContentLoaded", function() {
   const registerForm = document.getElementById("registerForm");
   const loginForm = document.getElementById("loginForm");
 
-  const successBanner = document.createElement("div");
-  // successBanner.id = "successBanner";
```
Comment (varshith257) [inline log/script.js:9]: Why JS Code has been deleted?

**Review by varshith257 — CHANGES_REQUESTED** [review]

**Comment by DiwareNamrata23** [comment]
@varshith257 is it working good now ?

**Comment by varshith257** [comment]
@Anjaliavv51 Once verify it

**Comment by Anjaliavv51** [comment]
@DiwareNamrata23 please check the deployment in netlify , their is gap between the sub footer section in footer tag and also the login container is overflowed.


Screenshot_20240810_224310_Chrome

**Comment by Anjaliavv51** [comment]
@varshith257 my suggestion is that their is no need of header and footer to the login page.

**Review by Anjaliavv51 — CHANGES_REQUESTED** [review]
@DiwareNamrata23 please check the deployment in netlify , their is gap between the sub footer section in footer tag and also the login container is overflowed.


Screenshot_20240810_224310_Chrome

**Comment by Anjaliavv51** [comment]
@varshith257 my suggestion is that their is no need of header and footer to the login page.

The current login page looks great , I this issue and PR need to be closed.

**Pull request closed without merging**